src/tagish/src/main/java/com/tagish/auth/DBLogin.java

```diff
@@ -182,6 +177,39 @@ private int getFailedLoginCount(Connection con, String username, AuditEventType
 
     }
 
+	private boolean accountIsLocked(Connection con, String username, AuditEventType event) {

```

Comment:
Is there really any reason to pass the event here given the intended functionality? Why not just reference the enum type directly?